public/app/plugins/panel/table/renderer.ts

```diff
@@ -217,12 +217,12 @@ export class TableRenderer {
 
     if (this.colorState.cell) {
       style = ' style="background-color:' + this.colorState.cell + ';color: white"';
+      cellClasses.push('white');
```

Comment:
class name white is not so good naming. 

Was thinking that class names could be convenient to use in general for table cells and not only for links, but that's maybe not a good idea - what do you think?

Was thinking of something like this
* table-panel-cell-color-cell
* table-panel-cell-color-row
* table-panel-cell-color-value

Then the `style: color: white` can be removed as well and instead be styled from sass. Exception is when table-panel-cell-color-value